.github/workflows/deploy-sdk.yml

```diff
@@ -0,0 +1,215 @@
+name: Deploy SDK
+
+on:
+  push:
+    branches: [main, v1] # all branches where deployments currently occur. Make sure this list matches list of branches in  `.releaserc` file. 
```

Comment:
@levibostian We still have `alpha` and `beta` in the .releaserc file. Should those be changed in this PR?